### PR TITLE
Coincheck Supplemental plugin

### PR DESCRIPTION
**Pull request by macanudo527**
Coincheck only really has an exchange for BTC at the moment. Everything else is sold on a marketplace which is not part of the REST API. You can't even retrieve transfers via the REST API, so this plugin just processes marketplace buys. It's incomplete, but I'm not sure how much longer Coincheck will be in business. They suffered a major attack a couple of years ago and haven't really recovered.

I no longer use them. This is just for a few minor old buys I have.

### CODE REVIEW

File: tests/test_plugin_coincheck.py
Comment (eprbell) [inline tests/test_plugin_coincheck.py:1]: Is this file a duplicate of tests/test_plugin_coincheck_supplemental.py?

**Review by eprbell — CHANGES_REQUESTED** [review]
We'll need to remove spurious files (binance, etc.) before merging. Also add documentation for this plugin.

**Comment by macanudo527** [comment]
At this point, I think it would just be easier to wait for #80 to get merged, and then I'll patch up these other pulls.

**Comment by macanudo527** [comment]
With the merge, this should be clean now.

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (eprbell) [inline src/dali/plugin/input/csv/coincheck_supplemental.py:73]: Curious about this logic: do some timestamps end with UTC and some not?

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (eprbell) [inline src/dali/plugin/input/csv/coincheck_supplemental.py:76]: This InTransaction is a buy with no fee (either fiat or crypto): is this correct? I know exchanges sometimes offer zero-fee buys but in my experience they are limited in number and when they're over fees start appearing.

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (macanudo527) [reply src/dali/plugin/input/csv/coincheck_supplemental.py:73]: All timestamps that I have seen end with UTC. I guess I just added that for redundancy. Should I take it out?

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (macanudo527) [reply src/dali/plugin/input/csv/coincheck_supplemental.py:76]: This is a marketplace buy and not a trade. This exchange sells crypto assets the same way you would exchange money at like Travelex. They have a wide spread ~5% and one price for buys and one for sells.

Coincheck only has a trading 'floor' for BTCJPY.

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (eprbell) [reply src/dali/plugin/input/csv/coincheck_supplemental.py:73]: I would suggest taking it out, if there is no use case for it.

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (eprbell) [reply src/dali/plugin/input/csv/coincheck_supplemental.py:76]: Thanks for the explanation: sounds good, then.

File: src/dali/plugin/input/csv/coincheck_supplemental.py
Comment (macanudo527) [reply src/dali/plugin/input/csv/coincheck_supplemental.py]: OK, I took it out.

**Review by eprbell — APPROVED** [review]
LGTM

**Pull request merged**
